internal/k8s/helpers.go

```diff
@@ -142,10 +142,6 @@ func IsObjectEqual(oldObj, newObj client.Object) (bool, error) {
 				apiequality.Semantic.DeepEqual(oldObj.Status, newObj.Status) &&
 				apiequality.Semantic.DeepEqual(oldObj.GetAnnotations(), newObj.GetAnnotations()), nil
 		}
-	case *core_v1.Endpoints:
```

Comment:
I got confused for a while why we did not have `EndpointSlices` covered here. For clarity we could add small comment in [`IsObjectEqual()`](https://github.com/projectcontour/contour/blob/7e2dcaf44c40f0f6543d46bfb774ee2b4d3920ef/internal/k8s/helpers.go#L100-L158) that `EndpointSlices` are not processed by the same handler and therefore they do not need to be considered here.

Reply:
👍🏽